build/Build.Pack.cs

```diff
@@ -193,22 +193,38 @@ void CreateLinuxPackages(string runtimeId)
 
             foreach (var runtimeId in RuntimeIds.Where(x => x.StartsWith("win")))
             {
-                var framework = runtimeId.Equals("win") ? NetFramework : NetCore;
+                switch (runtimeId)
+                {
+                    case "win":
+                        PackWindowsZip(NetFramework, runtimeId);
+                        break;
+                    case "win-x86":
+                    case "win-x64":
+                        PackWindowsZip(NetCoreWindows, runtimeId);
+                        PackWindowsZip(NetCore, runtimeId);
+                        break;
+                    default:
+                        PackWindowsZip(NetCore, runtimeId);
```

Comment:
when would we go down this path? @acodrington

Reply:
win-arm64?